mimic/rest/cloudfeeds.py

```diff
@@ -0,0 +1,116 @@
+# -*- test-case-name: mimic.test.test_loadbalancer -*-
+"""
+Defines the control plane API endpoints for the Cloudfeeds Plugin.
+"""
+from uuid import uuid4
+from six import text_type
+from zope.interface import implementer
+from twisted.web.server import Request
+from twisted.plugin import IPlugin
+from mimic.catalog import Endpoint, Entry
+from mimic.imimic import IAPIMock
+from mimic.rest.mimicapp import MimicApp
+
+
+from characteristic import attributes
+
+
+Request.defaultContentType = 'application/json'
```

Comment:
Fixed in db0fa7c

On Tue, Jun 23, 2015 at 11:50 AM, Lekha Jeevan <notifications@github.com>
wrote:

> In mimic/rest/cloudfeeds.py
> <https://github.com/rackerlabs/mimic/pull/331#discussion_r33077260>:
>
> > +Defines the control plane API endpoints for the Cloudfeeds Plugin.
> > +"""
> > +from uuid import uuid4
> > +from six import text_type
> > +from zope.interface import implementer
> > +from twisted.web.server import Request
> > +from twisted.plugin import IPlugin
> > +from mimic.catalog import Endpoint, Entry
> > +from mimic.imimic import IAPIMock
> > +from mimic.rest.mimicapp import MimicApp
> > +
> > +
> > +from characteristic import attributes
> > +
> > +
> > +Request.defaultContentType = 'application/json'
>
> This is setting the global state and should not be set. Refer #202
> <https://github.com/rackerlabs/mimic/issues/202>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rackerlabs/mimic/pull/331/files#r33077260>.
>



-- 
Samuel A. Falvo II